src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -583,13 +583,13 @@ private void ConfigureGradientBrush(ref PaintWrapper paintWrapper, Size targetSi
                     var center = radialGradient.Center.ToPixels(targetSize).ToSKPoint();
                     var radius = (float)(radialGradient.Radius * targetSize.Width);
 
-                    // TODO: There is no SetAlpha in SkiaSharp
-                    //paint.setAlpha(128);
+                    var origin = radialGradient.GradientOrigin.ToPixels(targetSize).ToSKPoint();
 
                     // would be nice to cache these shaders possibly?
-                    using (var shader =
-                        SKShader.CreateRadialGradient(center, radius, stopColors, stopOffsets, tileMode))
-                    {
+                    using (var shader = SKShader.CreateCompose(
```

Comment:
1) LINQ usage in hot code path
2) we should probably still use CreateRadialGradient when gradient origin is not set

Reply:
I've removed the Linq from the PR, though it is still used in this method on lines above. I'm happy to fix that if you prefer.

I've added a comparison to use the normal RadialGradient renderer as it was before, when the origin is the same as the center.

I also notice an issue with the reference point of the stops not being reversed so I've fixed that and added a new test.